homeassistant/components/voip/config_flow.py

```diff
@@ -36,6 +41,10 @@ async def async_step_user(
                 step_id="user",
             )
 
+        if CONF_SIP_USER in user_input and is_none_or_empty(
```

Comment:
This is not necessary for the main flow, as we don't ask for any values. The user is only set in the options flow.